model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer_config.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from typing import List, Dict
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+
+
+class BaseQuantizerConfig(object):
```

Comment:
No point in defining a base (abstract) class that does nothing that's common to the classes that inherit it.

Reply:
> No point in defining a base (abstract) class that does nothing that's common to the classes that inherit it.

Not sure I agree, if you want to be able to pass an argument to a function of type BaseQuantizerConfig and only infer its actual type during runtime, then defining an abstract base class and inherit from it is a good practice

Reply:
okay. thanks. indeed it can be dropped, however, it may be better practice to keep it.
I turned it to abstract class since I need this abstraction for common BaseTrainableQuantizer class.